lib/berlim/accounts/accounts.ex

```diff
@@ -28,4 +33,32 @@ defmodule Berlim.Accounts do
   def change_admin(%Admin{} = admin) do
     Admin.changeset(admin, %{})
   end
+
+  def list_taxis do
+    Repo.all(
+      from t in Taxi,
+        order_by: [asc: :smtt],
+        select: t
```

Comment:
pode omitir o select, não fará diferença, basta `from(t in Taxi, order_by: smtt)`, afinal `asc` é o default também